test/check-application

```diff
@@ -1139,6 +1139,85 @@ class TestApplication(testlib.MachineCase):
         self.allow_journal_messages(".*podman/podman.sock/.*: couldn't connect:.*")
         self.allow_journal_messages(".*podman/podman.sock/.*/events.*: received truncated HTTP response.*")
 
+    @testlib.nondestructive
+    def testCreateContainerSystem(self):
+        self._testCreateContainer(True)
+
+    @testlib.nondestructive
+    def testCreateContainerUser(self):
+        self._testCreateContainer(False)
+
+    def _testCreateContainer(self, auth):
+        self.execute(True, "podman run -d -p 5000:5000 --name registry registry:2")
+        self.execute(True, "podman run -d -p 6000:5000 --name registry_alt registry:2")
+        # Add local insecure registry into registries conf
+        self.execute(True, "echo \"{0}\" > /etc/containers/registries.conf && systemctl stop podman.service".format(REGISTRIES_CONF))
+        # Push busybox image to the local registries
+        self.execute(True, "podman tag quay.io/libpod/busybox localhost:5000/my-busybox && podman push localhost:5000/my-busybox")
+        self.execute(True, "podman tag quay.io/libpod/busybox localhost:6000/my-busybox && podman push localhost:6000/my-busybox")
+        # Untag busybox image which duplicates the image we are about to download
+        self.execute(True, "podman rmi -f quay.io/libpod/busybox localhost:5000/my-busybox localhost:6000/my-busybox")
+
+        self.login(auth)
+
+        b = self.browser
+        container_name = "busybox-downloaded"
+        # No local results found
+        b.wait_visible("#containers-containers button.pf-c-button.pf-m-primary:not([disabled])")
+        b.click("#containers-containers button.pf-c-button.pf-m-primary")
```

Comment:
Why doing this? `b.click()` already waits for element to be visible and not disabled.